components/src/commonMain/kotlin/com/imashnake/animite/components/media/MediaSmall.kt

```diff
@@ -59,19 +51,12 @@ fun <T> MediaSmallRow(
                 style = MaterialTheme.typography.titleMedium,
                 modifier = Modifier
                     .padding(start = LocalPaddings.current.large)
-                    .landscapeCutoutPadding()
             )
         }
         LazyRow(
             horizontalArrangement = Arrangement.spacedBy(LocalPaddings.current.small),
             contentPadding = PaddingValues(
-                start = LocalPaddings.current.large + if (
-                    LocalConfiguration.current.orientation == Configuration.ORIENTATION_LANDSCAPE
-                ) {
-                    WindowInsets.displayCutout.asPaddingValues()
-                        .calculateLeftPadding(LayoutDirection.Ltr)
-                } else 0.dp,
-                end = LocalPaddings.current.large
```

Comment:
but landscape mode :/

Reply:
Handle your insets properly :weary:

Reply:
i am............ check discord

Reply:
Since you never sent anything, I'm gonna resolve this
I think insets in general are something we need to figure out properly, whether that's before or after this is up for discussion :P